### DISCUSSION TITLE
Problem with DRM removal using Calibre

### DISCUSSION THREAD

**Discussion by Lisaha1** [apprenticeharper/DeDRM_tools#2243] [General]
I have successfully configured the DeDRM plugin (v10) in Calibre and added the KFX Input plugin but I am getting the message 'Calibre cannot covert this book is locked by DRM' I am using Kindle for Mac V1.30. 

Any help with this would be much appreciated.

**Comment by ElleKayEm** [comment]
Be sure to disable KFX downloads.  See method 2B here: https://www.mobileread.com/forums/showthread.php?t=283371

And some Mac specific notes here: https://www.mobileread.com/forums/showpost.php?p=3819708&postcount=508
